aiida/backends/tests/export_and_import.py

```diff
@@ -1481,6 +1483,175 @@ def get_all_node_links(self):
                   edge_project=['label', 'type'], output_of='input')
         return qb.all()
 
+    class ComplexGraph:
+        """
+        A class responsible to create a"complex" graph with all available link types
+        (INPUT, CREATE, RETURN and CALL)
+        """
+
+        test_case = None
```

Comment:
should these be class variables?
if they are just instance variables, you can remove these two lines

Reply:
I prefer to keep them for clarity - I don't like the approach of using undefined variables even if Python allows it - I find it dirty

Reply:
But I will document them though